playground/mocks/idp/idx/data/select-factor-verification-for-recovery.json

```diff
@@ -0,0 +1,111 @@
+{
+  "stateHandle": "02h50hMLvmuZUuoKCShHKZytlDeFRnn8KG-rcd8Ay5",
+  "version": "1.0.0",
+  "expiresAt": "2019-09-25T23:55:18.000Z",
+  "step": "AUTHENTICATE",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "select-factor",
+        "href": "http://localhost:3000/idp/idx/challenge",
+        "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
+        "value": [
+            {
+              "name": "factorId",
+              "type": "set",
+              "options": [
+                {
+                  "label": "Password Label",
+                  "value": "00u2j17ObFUsbGfLg0g4"
+                },
+                {
+                  "label": "Email Label",
+                  "value": "emf2j1ccd6CF4IWFY0g3"
+                }
+              ]
+            },
+            {
+              "name": "stateHandle",
+              "required": true,
+              "value": "02h50hMLvmuZUuoKCShHKZytlDeFRnn8KG-rcd8Ay5",
+              "visible": false,
+              "mutable": false
+            }
+        ]
+      }
+    ]
+  },
+  "factors": {
+    "type": "array",
+    "value": [
+      {
+        "factorType": "password",
+        "factorId": "00u2j17ObFUsbGfLg0g4"
+      },
+      {
+        "factorType": "email",
+        "factorId": "emf2j1ccd6CF4IWFY0g3"
+      }
+    ]
+  },
+  "user": {
+    "type": "object",
+    "value": {
+      "id": "00u2j17ObFUsbGfLg0g4"
+    }
+  },
+  "cancel": {
+    "rel": [
+      "create-form"
+    ],
+    "name": "cancel",
+    "href": "http://localhost:3000/idp/idx/cancel",
+    "method": "POST",
+    "accepts": "application/vnd.okta.v1+json",
+    "value": [
+      {
+        "name": "stateHandle",
+        "required": true,
+        "value": "02h50hMLvmuZUuoKCShHKZytlDeFRnn8KG-rcd8Ay5",
+        "visible": false,
+        "mutable": false
+      }
+    ]
+  },
+  "context": {
+    "rel": [
+      "create-form"
+    ],
+    "name": "context",
+    "href": "http://localhost:3000/idp/idx/context",
+    "method": "POST",
+    "accepts": "application/vnd.okta.v1+json",
+    "value": [
+      {
+        "name": "stateHandle",
+        "required": true,
+        "value": "02h50hMLvmuZUuoKCShHKZytlDeFRnn8KG-rcd8Ay5",
+        "visible": false,
+        "mutable": false
+      }
+    ]
+  },
+  "recoveryFactor" : {
+    "type" : "object",
+    "value" : {
+      "factorType": "password",
```

Comment:
can I recovery `password` by choosing `password` (`factors` array has password factor) ?

Reply:
![idx-demo3](https://user-images.githubusercontent.com/17267130/73553544-fd617c00-43fe-11ea-8a46-58e89e0aca40.gif)


This is the flow.

Reply:
the mock data doesn't reflect your flow. `select-factor` form has two factor (password and email) when you trying to recovery password.